client/package.json

```diff
@@ -3,11 +3,11 @@
   "version": "1.0.0-beta.2",
   "private": true,
   "scripts": {
-    "start": "lerna run --scope @openmsupply-client/* --parallel start",
-    "build": "lerna run --scope @openmsupply-client/* build",
-    "serve": "lerna run --scope @openmsupply-client/* --parallel serve",
-    "clean": "lerna run --scope @openmsupply-client/* --parallel clean",
-    "compile": "lerna run --scope @openmsupply-client/* --parallel tsc --since HEAD",
+    "start": "lerna run --scope @openmsupply-client/host --parallel start",
```

Comment:
I would assume original intent was to user federation module to server all packages as microfront ends, thus the `*`, had to remove it so that electron can also be started from the root while still having normal `start` script when running from electron package